index.js

```diff
@@ -207,18 +207,26 @@ function init (ssb, config) {
     if (!isGroup(content.recps[0])) return publish.apply(this, args)
 
     onKeystoreReady(() => {
-      getGroupTangle(content.recps[0], (err, tangle) => {
-        // NOTE there are two ways an err can occur in getGroupTangle
-        // 1. recps is not a groupId
-        // 2. unknown groupId,
-
-        // Rather than cb(err) here we we pass it on to boxers to see if an err is needed
+      getGroupTangle(content.recps[0], 'group', (err, groupTangle) => {
         if (err) return publish.apply(this, args)
+        getGroupTangle(content.recps[0], 'members', (err, membersTangle) => {
+          // NOTE there are two ways an err can occur in getGroupTangle
+          // 1. recps is not a groupId
+          // 2. unknown groupId,
+
+          // Rather than cb(err) here we we pass it on to boxers to see if an err is needed
+          // TODO: why do we skip adding the tangles if we err? we should skip publishing in that case
```

Comment:
todo. was there some big reason behind this choice or is it fine if i change it @mixmix ?

Reply:
changed in fc7a2c0